File: pgm/ABC_Dart/ABC_419_A.dart
import 'dart:collection';
import 'dart:io';
// import 'dart:math';

void main() {
  final String S = stdin.readLineSync()!;
  final String red = 'red';
  final String blue = 'blue';
  final String green = 'green';
  final String SSS = 'SSS';
  final String FFF = 'FFF';
  final String MMM = 'MMM';
  final String Unknown = 'Unknown';
  final Map<String, String> Language = Map.unmodifiable(
    HashMap<String, String>.from({red: SSS, blue: FFF, green: MMM}),
  );
  print(Language.containsKey(S) ? Language[S] : Unknown);
  exit(0);
}

File: pgm/ABC_PHP/ABC_419_A.php
<?php
$stream = fopen('php://stdin', 'r');
$line = fgets($stream);
fclose($stream);
const Unknown = 'Unknown';
const red = 'red';
const blue = 'blue';
const green = 'green';
const SSS = 'SSS';
const FFF = 'FFF';
const MMM = 'MMM';
$S = trim($line);
$language = [
    red => SSS,
    blue => FFF,
    green => MMM,
];
echo ($language[$S] ?? Unknown) . PHP_EOL;
exit(0);
